org.civicrm.reqangular/test/models/instances/relationship.instance.spec.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/lodash',
+  'common/mocks/data/relationship.data',
+  'common/angularMocks',
+  'common/models/instances/relationship.instance'
+], function (_, relationshipData) {
+  'use strict';
+
+  describe('RelationshipInstance', function () {
+    var ModelInstance, RelationshipInstance;
+
+    beforeEach(module('common.models.instances'));
+    beforeEach(inject(function (_ModelInstance_, _RelationshipInstance_) {
+      ModelInstance = _ModelInstance_;
+      RelationshipInstance = _RelationshipInstance_;
+    }));
+
+    it('inherits from ModelInstance', function () {
+      expect(_.functions(RelationshipInstance))
+        .toEqual(jasmine.arrayContaining(_.functions(ModelInstance)));
+    });
+
+    describe('isValid()', function () {
+      beforeEach(function () {
+        _.forEach(relationshipData.named, function (relationship, index) {
```

Comment:
Please create a new object `instances` inside the forEach. And test using `instances.isActive` or similar. Right now from the `it` blocks, it looks like you are testing the mock data and not the instances.